colourgame.html

```diff
@@ -7,29 +7,40 @@
   </body>
 <script>
 var guess,
-  answer,
-  clue1,
-  clueguess;
+  	answer,
+  	response,
+  	color,
+  	number;
+
+number = Math.floor(Math.random() * 10);
+
+if (number < 3) {
+  color = "Red";
+
+} else if (number > 3 && number < 7) {
```

Comment:
the values 3 and 7 aren't accounted for.
you could fix it by changing this to `(number >= 3 && number <= 7)`